gnucash/ui/gnc-plugin-page-owner-tree.ui

```diff
@@ -170,6 +170,11 @@
         <attribute name="action">GncPluginPageOwnerTreeActions.OTProcessPaymentAction</attribute>
         <attribute name="tooltip" translatable="yes">Process Payment</attribute>
       </item>
+      <item>
+        <attribute name="label" translatable="yes">Customer's Invoices</attribute>
+        <attribute name="action">GncPluginPageOwnerTreeActions.OTSearchInvoicesAction</attribute>
+        <attribute name="tooltip" translatable="yes">Customer's Invoices</attribute>
```

Comment:
The menu item label and tooltip are respectively misleading and redundant. The former should be "Find Customer Invoice…" (the ellipsis indicates that the menu item opens a dialog that requires further input to do anything useful) and the tooltip should be something like "Open the Find Invoice Dialog with the invoices already filtered for this customer."